### PR TITLE
Add lots of contributors

### LINKED ISSUE #992: all-contributors requests for the book dash, eScience center members, CW20 and more
### Summary

* Contributors to add:
- [x] @mkuzak for bug, review, content
- [x] @c-martinez for bug, review content
- [x] @bouweandela for content
- [x] @PabRod for content
- [x] @sverhoeven for comtent
- [x] @eirini-zormpa for bug
- [x] @dblana for content
- [x] @orchid00 for review
- [x] @nlharris for review
- [x] @FrancesMadden for content
- [x] @chadsgilbert for bug
- [x] @sdruskat for content
- [x] @fmcooper for content
- [x] @JoannaLeng for content
- [x] @colinsauze for idea
- [x] @Chrisisour for idea
- [x] @kessonovitch for idea (content under prep)
- [x] @ArielleBL (content under prep)
- [x] @kdixey for idea (content under prep)
- [ ] @nalessandroni for idea (content under prep)
- [ ] @alexwlchan for idea (content under prep)
- [ ] @npch  for idea (content under prep)
- [ ] @trotts for idea (Newcastle)
- [ ] @vladoxNCL for idea (Newcastle)
- [ ] @ppintosilva for idea (Newcastle)
- [ ] @SedarOlmez94 for idea (Newcastle)
- [ ] @rosesisk for idea (Newcastle)
- [ ] @mednche for idea (Newcastle)
- [ ] @paulbaniqued for idea (Newcastle)
- [ ] @GeorgiaA for idea (Newcastle)
- [ ] @davehedgehog for idea (Newcastle)
- [ ] @annabelelizabethwhipp for idea (Newcastle)
- [ ] @sianbladon for idea (Newcastle)
- [ ] @watson-c for idea (Newcastle)
- [ ] @philipdarke for idea (Newcastle)

### What needs to be done?

- [ ] Call all-contributors bot
- [ ] Check is everyone has been added

### Who can help?

* all-contributors bot 

---

### Updates

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

### PR DESCRIPTION
**Pull request by KirstieJane**
### Summary

Fixes #992

I got the All Contributors command line interface working so I a) added all the remaining folks in #992 in one batch AND the table is now ordered in alphabetical order 😍 

### List of changes proposed in this PR (pull-request)

* Add all participants in #992 
* Table formatted beautifully 😍 (I already did that but I like it A LOT!)

### What should a reviewer concentrate their feedback on?

- [ ] Is everyone added? Are we missing any additional contributions?
- [ ] Everything looks ok?

There are a few "real" names I don't know. If we do know them then I think we should add them, but if we don't know them then we can just leave them as they are. My rubric is that if I can manually find their name from their github user page or if they have attended an event then I think its ok to add their full name....happy to take a different route though!

- [ ] @kir0ul 
- [ ] @m-rivera 
- [ ] @philipda 
- [ ] @watson-c 
- [ ] @davehedgehog

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by malvikasharan** [comment]
I know the remaining names here, I can add them if you don't merge this branch - or will open another PR.

**Comment by KirstieJane** [comment]
@malvikasharan - did you double check that I hadn't missed anyone from #992 ?

**Comment by KirstieJane** [comment]
Oh yeah - tell me the names and I'll add them now 😄

**Comment by malvikasharan** [comment]
> @malvikasharan - did you double check that I hadn't missed anyone from #992 ?

I haven't double checked.

**Comment by malvikasharan** [comment]
@kir0ul: Andrea Pierré
 @m-rivera: Can't remember this one
 @philipdarke: Philip Darke
 @watson-c: Charlotte Watson

**Comment by KirstieJane** [comment]
Ah - I think we've added @philipda instead of @philipdarke - should I remove them @malvikasharan?

(Sorry for the mistake!)

**Comment by malvikasharan** [comment]
All the Newcastle students:

- Cameron Trotter / @trotts
- Vlad G. Zelaya / vladoxNCL
- Pedro / ppintosilva
- Sedar Olmez / SedarOlmez94
- Rose Sisk / rosesisk
- Natacha Chenevoy / mednche
- Paul Baniqued / paulbaniqued
- Georgia Atkinson/ GeorgiaA
- Tess Gough /davehedgehog
- Annabel Whipp /annabelelizabethwhipp
- Sian Bladon / sianbladon
- Charlotte Watson / watson-c
- Philip Darke / philipdarke

File: .all-contributorsrc [hunk 1]
```diff
@@ -1040,6 +1040,141 @@
...
+      "login": "watson-c",
+      "name": "watson-c",
+      "avatar_url": "https://avatars3.githubusercontent.com/u/48691807?v=4",
+      "profile": "https://github.com/watson-c",
+      "contributions": [
+        "ideas"
+      ]
+    },
+    {
+      "login": "philipda",
```
Comment (malvikasharan) [inline .all-contributorsrc]:
```suggestion
      "login": "philipdarke",
```

File: .all-contributorsrc [hunk 2]
```diff
@@ -1040,6 +1040,141 @@
...
+      "name": "philipda",
```
Comment (malvikasharan) [inline .all-contributorsrc]:
```suggestion
      "name": "Philip Darke",
```

File: .all-contributorsrc [hunk 3]
```diff
@@ -1040,6 +1040,141 @@
...
+      "name": "Cameron Trotter",
+      "avatar_url": "https://avatars3.githubusercontent.com/u/9531671?v=4",
+      "profile": "http://trotts.io",
+      "contributions": [
+        "ideas"
+      ]
+    },
+    {
+      "login": "vladoxNCL",
+      "name": "vladoxNCL",
```
Comment (malvikasharan) [inline .all-contributorsrc]:
```suggestion
      "name": "Vlad G. Zelaya",
```

File: README.md [hunk 4]
```diff
@@ -208,27 +217,39 @@ Thanks goes to these wonderful people ([emoji key](https://github.com/all-contri
...
     <td align="center"><a href="https://github.com/nathanbegbie"><img src="https://avatars0.githubusercontent.com/u/7100966?v=4" width="100px;" alt=""/><br /><sub><b>nathanbegbie</b></sub></a><br /><a href="https://github.com/alan-turing-institute/the-turing-way/issues?q=author%3Anathanbegbie" title="Bug reports">🐛</a> <a href="#ideas-nathanbegbie" title="Ideas, Planning, & Feedback">🤔</a></td>
     <td align="center"><a href="https://github.com/nbarlowATI"><img src="https://avatars3.githubusercontent.com/u/33832774?v=4" width="100px;" alt=""/><br /><sub><b>nbarlowATI</b></sub></a><br /><a href="https://github.com/alan-turing-institute/the-turing-way/issues?q=author%3AnbarlowATI" title="Bug reports">🐛</a> <a href="#content-nbarlowATI" title="Content">🖋</a></td>
+    <td align="center"><a href="https://github.com/philipda"><img src="https://avatars0.githubusercontent.com/u/32183281?v=4" width="100px;" alt=""/><br /><sub><b>philipda</b></sub></a><br /><a href="#ideas-philipda" title="Ideas, Planning, & Feedback">🤔</a></td>
```
Comment (malvikasharan) [inline README.md]:
```suggestion
    <td align="center"><a href="https://github.com/philipda"><img src="https://avatars0.githubusercontent.com/u/32183281?v=4" width="100px;" alt=""/><br /><sub><b>philipdarke</b></sub></a><br /><a href="#ideas-philipdarke" title="Ideas, Planning, & Feedback">🤔</a></td>
```

Comment (malvikasharan) [inline README.md]:
```suggestion
    <td align="center"><a href="https://github.com/philipdarke"><img src="https://avatars0.githubusercontent.com/u/32183281?v=4" width="100px;" alt=""/><br /><sub><b>philipda</b></sub></a><br /><a href="#ideas-philipdarke" title="Ideas, Planning, & Feedback">🤔</a></td>
```

**Comment by malvikasharan** [comment]
I double checked. Looks like I git them all.

**Pull request merged**